cezo_fl/aggregation.py

```diff
@@ -0,0 +1,34 @@
+import torch
+
+
+def mean(num_sample_clients: int, local_grad_scalar_list):
```

Comment:
nit: we should add type annotation as much as possible. For example here should be

```python
def mean(num_sample_clients: int, local_grad_scalar_list: list[list[torch.Tensor]]) -> list[torch.Tensor]:
```
`local_grad_scalar_list` -- is a list of sampled clients of a list of local updated tensors.

I wonder do we really need the `num_sample_clients`? I think we can get it from the `each_client_update`.

Reply:
resolved